Updated feature list

shivarammysore opened the pull request.

anarkiwi commented:
" Policy based forwarding
￼ +- 802.1x authentication support"

Suggest to say instead:

 policy based forwarding to offload processing to external systems (Eg 802.1x via hostapd).

Makes it clearer that 802.1x is not and does not need to be in the controller.

shivarammysore commented:
Good suggestion.  Fixed and updated Pull request.

Thanks!

On Tue, Aug 2, 2016 at 1:12 PM, Josh Bailey <notifications@github.com>
wrote:

> " Policy based forwarding
> ￼ +- 802.1x authentication support"
>
> Suggest to say instead:
>
> policy based forwarding to offload processing to external systems (Eg
> 802.1x via hostapd).
>
> Makes it clearer that 802.1x is not and does not need to be in the
> controller.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/REANNZ/faucet/pull/147#issuecomment-237028683>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/ABmf84HcwVefsqVtc1iijLv-Jb9aCPjqks5qb6SzgaJpZM4JZ3M6>
> .
>
